drivers/openstack/openstack.go

```diff
@@ -555,7 +565,7 @@ func (d *Driver) resolveIds() error {
 		}).Debug("Found image id using its name")
 	}
 
-	if d.FloatingIpPool != "" {
+	if d.FloatingIpPool != "" && d.IpPoolType != "nova" {
```

Comment:
`d.FloatingIpPool` is set explicitly to `nova` when it's empty and the `d.IpPoolType == "nova"` (ln 277), so this check is unnecessary